cmd/openqa-mon/util.go

```diff
@@ -94,16 +73,9 @@ func homeDir() string {
 	return usr.HomeDir
 }
 
-func createIntRange(min int, max int, offset int) []int {
-	ret := make([]int, 0)
-	for i := min; i <= max; i++ {
-		ret = append(ret, i+offset)
-	}
-	return ret
-}
-
 func createInt64Range(min int64, max int64, offset int64) []int64 {
-	ret := make([]int64, 0)
+	// create with capacity to avoid runtime reallocation
+	ret := make([]int64, 0, 1+max-min)
```

Comment:
```suggestion
	ret := make([]int64, 0, (max-min)+1)
```
Functionally the same, but people are used to this notation when counting the total numbers within a range. Do you agree?